src/DecayTimeFinderAction.cpp

```diff
@@ -55,4 +60,11 @@ G4double DecayTimeFinderAction::GetPositronRange()
 {
   G4double d2 = pow(m_annihilationX-m_radDecayX, 2) + pow(m_annihilationY-m_radDecayY, 2) + pow(m_annihilationZ-m_radDecayZ, 2);
   return sqrt(d2);
+}
+
+void DecayTimeFinderAction::NewStage()
```

Comment:
I'm not sure exactly what the name of this function means, but more importantly, I don't think you call it from anywhere and so you'll never get the output.